test/test_openapi.py

```diff
@@ -178,6 +185,24 @@ def route1():
 
         self.assertEqual(app.spec()['tags'], [{'name': 'foo'}, {'name': 'qux'}])
 
+    def _expected_oauth_spec(self, app_name: str):
```

Comment:
This is essentially duplicating the code under test. Extract that code under test into a method/function and patch method to return "nothing" for these tests.